qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -143,3 +146,4 @@
             gagent_serial_type = isa
             serials += " org.qemu.guest_agent.0"
             gagent_start_cmd = "pgrep qemu-ga || qemu-ga -d -m isa-serial -p /dev/ttyS1"
+            gagent_status_cmd = "pgrep qemu-ga"
```

Comment:
I saw the test steps in polarion is "#systemctl status qemu-guest-agent" after init 3. The status should be "Active: active (running)". What I concern is, will "pgrep qemu-ga" have the same effect with "systemctl"? And I saw that in other sub cases, pgrep is also used to check running status..